vantage6/cli/node.py

```diff
@@ -309,13 +309,16 @@ def cli_node_start(name, config, environment, system_folders, image, keep,
     info("Creating Docker data volume")
     data_volume = docker_client.volumes.create(
         f"{ctx.docker_container_name}-vol")
```

Comment:
If you are already at it.. change this one too: 
`ctx.docker_volume_name`

Reply:
Done